security/firely-auth/firely-auth-sso.rst

```diff
@@ -56,12 +73,24 @@ Configuring a new client application in Azure Active Directory (Azure AD) using
     - Select "Token configuration"
     - Select "+ Add optional claim"
     - Select "ID token"
-    - Choose a claim based on which the remote SSO account should be matched against a :ref:`password-less local account <firely_auth_mgmt_sso_user>`. Currently, Firely Auth only supports the "email" claim, but additional  claims may be supported in future. Please make sure to add this claim to the locally provisioned user account.
+    - Choose a claim based on which the remote SSO account should be matched against a :ref:`password-less local account <firely_auth_mgmt_sso_user>`. Currently, Firely Auth supports the ``email`` and ``oid`` claim. Please make sure to add at least one of these claims to the locally provisioned user account.
+    - The ``oid`` claim is populated based on the Object ID that can be found on each user profile in Entra ID. Matching on ``oid`` claim is preferred over the ``email`` claim as it is guaranteed not to change.
+
+#. Configure the ``API permissions`` section
+
+    - Select "API permissions"
+    - Make sure to at least add "email", "profile", and "profile" as permissions
 
 #. Configure the ``ExternalIdentityProviders`` section
 
     - Select "Overview".
     - Select "Endpoints"
     - Use the base url of any of the displayed OAuth 2.0 endpoints as the authority in the settings. It should start with "https://login.microsoftonline.com/" followed by the tenant id within Microsoft Entra ID, ending in "/v2.0/".
```

Comment:
> Use the base url of any of the displayed OAuth 2.0 endpoints as the authority in the settings

This isn't true - my metadata endpoint is `https://login.microsoftonline.com/db0b1140-e3e3-4719-866d-9c6eaffaacec/v2.0/.well-known/openid-configuration` but all other endpoints have `oauth2` in the middle, such as `https://login.microsoftonline.com/db0b1140-e3e3-4719-866d-9c6eaffaacec/oauth2/v2.0/token`. 

So the FA setting requires `https://login.microsoftonline.com/db0b1140-e3e3-4719-866d-9c6eaffaacec/v2.0/` in my case. I don't know the best way to describe this. It looks like the value of the `Authority` setting has to be `https://login.microsoftonline.com/<Directory (tenant) ID of the registered application>/v2.0`